protox/Cargo.toml

```diff
@@ -48,6 +48,7 @@ prost-reflect = { version = "0.12.0", features = ["miette", "text-format"] }
 prost-types = "0.12.1"
 protox-parse = { version = "0.5.0", path = "../protox-parse" }
 thiserror = "1.0.50"
+anyhow = "1.0.79"
```

Comment:
The `protox::Error` type is already intended to contain arbitrary payloads through `protox::Error::new()`, we shouldn't need a dependency on `anyhow`